### ISSUE TITLE
Certain views are unusually slow to load

### ISSUE THREAD

**Issue by ebugden** [theia-ide/theia-trace-extension#405]
Slowness varies between user (ex. 40 s for @MatthewKhouzam, slightly longer for @arfio). @tahini suspects this is unusual and worth more exploration. Views to look into:

* Irq Analysis – latency vs time
* Futex contention analysis- latency vs time
* System call latency – latency vs time
* Flame chart 

More context in #296

**Comment by arfio** [comment]
It depends a lot on the trace being analyzed. In my case, it was pretty slow but the trace was >1GB. There should be some way of having a progress bar at least.

**Comment by ebugden** [comment]
> It depends a lot on the trace being analyzed. In my case, it was pretty slow but the trace was >1GB. There should be some way of having a progress bar at least.

Good point! I added that this was not urgent and that @thefinaljob is working on adding a progress bar #284. 

I created the issue to have a record, but we can close it if eventually we realize it was just the file size.

**Comment by ebugden** [comment]
All of the latency vs. time views are currently filtered out client-side (#508) since these views contain discrete data and scatter plots aren't supported yet #45
